<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>931</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit_Status">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>811</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Interface">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>46</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>75</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>75</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>811</width>
      <height>421</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Pkt num</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Distance</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tx power</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>RSSI</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>67</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_distance">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_copyTable">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clearTable">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear table</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Power:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetPower">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxPower">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>59</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-15</number>
    </property>
    <property name="maximum">
     <number>7</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>7</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
